src/c/errorlist.h

```diff
@@ -24,7 +24,6 @@
 #define EDGEX_REMOTE_SERVER_DOWN (devsdk_error){ .code = 15, .reason = "Remote server unresponsive" }
 #define EDGEX_PROFILE_PARSE_ERROR (devsdk_error){ .code = 16, .reason = "Error while parsing device profile" }
 #define EDGEX_HTTP_CONFLICT (devsdk_error){ .code = 17, .reason = "HTTP 409 Conflict" }
-#define EDGEX_CONSUL_RESPONSE (devsdk_error){ .code = 18, .reason = "Unable to process response from consul" }
```

Comment:
Make this EDGEX_REGISTRY_RESPONSE and open an issue for making `edgex_keeper_get_tree` set this error when it fails?